gui/gui/src/TGView.cxx

```diff
@@ -478,11 +481,15 @@ void TGView::ScrollCanvas(Int_t new_top, Int_t direction)
       points[0].fX = points[3].fX = 0;
       points[1].fX = points[2].fX = fCanvas->GetWidth();
       xsrc = xdest = 0;
+      #ifndef R__HAS_COCOA
       cpywidth = 0;
+      #endif
```

Comment:
As discussed above, we should be able to remove this if we initialize it to `0` in the declaration (but double-check, please).
```suggestion
```